### DISCUSSION TITLE
Are you interested in becoming a listed KEDA end-user?

### DISCUSSION THREAD

**Discussion by tomkerkhove** [spaceuptech/space-cloud#1582] [Q&A]
Are you interested in becoming a listed KEDA end-user?

**Comment by YourTechBud** [comment]
Absolutely. Keda is core to SpaceCloud and we would love to be a part of the Keda Community.

Let me know how we can proceed.

**Reply by tomkerkhove** [reply]
That's great, thank you! Are you willing to open a PR similar to https://github.com/kedacore/keda-docs/pull/437 please?

**Reply by YourTechBud** [reply]
Absolutely. Will do it right away.
